src/sentry/static/sentry/app/components/acl/featureDisabled.jsx

```diff
@@ -0,0 +1,119 @@
+import {Box, Flex} from 'grid-emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {t, tct} from 'app/locale';
+import Alert from 'app/components/alert';
+import Button from 'app/components/button';
+import ExternalLink from 'app/components/externalLink';
+
+const CONFIG_DOCS_URL = 'https://docs.sentry.io/server/config/';
+
+/**
+ * DisabledInfo renders a componet informing that a feature has been disabled.
+ *
+ * By default this component will render a help button which toggles more
+ * information about why the feature is disabled, showing the missing feature
+ * flag and linking to documentation for managing sentry server feature flags.
+ */
+class FeatureDisabled extends React.Component {
+  static propTypes = {
+    feature: PropTypes.string,
+    featureName: PropTypes.string,
+    hideHelpToggle: PropTypes.bool,
+    alert: PropTypes.bool,
+  };
+
+  state = {
+    showHelp: false,
+  };
+
+  toggleHelp = e => {
+    e.preventDefault();
+    this.setState({showHelp: !this.state.showHelp});
+  };
+
+  render() {
+    const {showHelp} = this.state;
+    const {feature, featureName, hideHelpToggle, alert} = this.props;
+
+    const featureDisabled = (
+      <React.Fragment>
+        <Flex justify="space-between">
+          {t('This feature is not enabled on your Sentry installation.')}
+          {!hideHelpToggle && (
+            <HelpButton
+              icon={this.showHelp ? 'icon-chevron-down' : 'icon-circle-info'}
```

Comment:
should this be `showHelp`?

```suggestion
              icon={showHelp ? 'icon-chevron-down' : 'icon-circle-info'}
```